assume/reinforcement_learning/learning_role.py

```diff
@@ -329,6 +367,9 @@ def compare_and_save_policies(self, metrics: dict) -> bool:
                         logger.info(
                             f"Stopping training as no improvement above {self.early_stopping_threshold} in last {self.early_stopping_steps} evaluations for {metric}"
                         )
+                        logger.info(
```

Comment:
maybe add the check if self.decay was true and display this info that there were these scheduelrs and maybe not the optimum was reached?